### DISCUSSION TITLE
Welcome to SETL Framework Discussions!

### DISCUSSION THREAD

**Discussion by qxzzxq** [SETL-Framework/setl#195] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by nttq1sub** [comment]
how can I config my hudi connector in main dir of SETL framework? 
and how can I skip test step when I run my project in SETL
hope to here from you soon
pls guide me. 
thanks so much
